concrete/tools/sitemap_search_selector.php

```diff
@@ -25,72 +28,72 @@
 */
 
 ?>
-<div class="ccm-ui" id="ccm-sitemap-search-selector">
 
-<?=Loader::helper('concrete/ui')->tabs(array(
-    array('sitemap', t('Full Sitemap')),
-    array('explore', t('Flat View')),
-    array('search', t('Search')),
-));
-?>
+<div class="ccm-ui" id="ccm-sitemap-search-selector">
 
-<div id="ccm-tab-content-sitemap" class="ccm-tab-content"></div>
+	<?php
+        echo $app->make('helper/concrete/ui')->tabs([
+            ['sitemap', t('Full Sitemap')],
+            ['explore', t('Flat View')],
+            ['search', t('Search')],
+        ]);
+    ?>
 
+	<div id="ccm-tab-content-sitemap" class="ccm-tab-content"></div>
 
-<div id="ccm-tab-content-explore" class="ccm-tab-content"></div>
+	<div id="ccm-tab-content-explore" class="ccm-tab-content"></div>
 
-<div id="ccm-tab-content-search" class="ccm-tab-content"></div>
+	<div id="ccm-tab-content-search" class="ccm-tab-content"></div>
 
 </div>
 
 <script type="text/javascript">
-ccm_sitemapSearchSelectorHideBottom = function() {
-	$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').hide();
-}
-
-ccm_sitemapSearchSelectorShowBottom = function() {
-	$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').show();
-}
-
+	ccm_sitemapSearchSelectorHideBottom = function() {
+		$('#ccm-sitemap-search-selector').parent().parent().find('.ui-dialog-buttonpane').hide();
```

Comment:
(I didn't know about that..) Just to be sure before I change it, @aembler any thought ?

Reply:
For PHP we adhere to [PSR](https://www.php-fig.org/psr/psr-2/#1-overview), I'm just assuming that it would also apply to e.g. JavaScript. If that's not the case, I guess we should update our coding style guide.

Reply:
@a3020 That makes sense, i'll make the changes 👍